### PR TITLE
fix: added ripple effect for voice commands

### LINKED ISSUE #2158: Add ripple effect when user clicks on the set of commands .
**Describe the feature you'd like**
The user is not prompted if the command is clicked or not. By mistake, I clicked the command twice or three not knowing if the command is clicked.

**Additional context**
Adding ripple effect on the commands will prompt the user that the command has been clicked

**Would you like to work on the issue?**
Yes

### PR DESCRIPTION
**Pull request by Bhavnaharitsa**
Fixes #2158 
Changes: Added ripple effect into the set of commands. 
It's just a replica of the earlier closed issue #2160 

Unfortunately, I had to close the pull request due to system crash.

### CODE REVIEW

File: app/src/main/res/drawable/ripple_effect.xml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
```
Comment (iamareebjamal) [inline app/src/main/res/drawable/ripple_effect.xml]: unnecessary

File: app/src/main/res/drawable-v21/ripple_effect.xml [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<ripple xmlns:android="http://schemas.android.com/apk/res/android"
```
Comment (iamareebjamal) [inline app/src/main/res/drawable-v21/ripple_effect.xml]: unnecessary

**Comment by iamareebjamal** [comment]
Why are you closing PRs and creating new ones?

**Comment by Bhavnaharitsa** [comment]
@iamareebjamal I have removed unnecessaries, but still, the circleci is failing. I have checked the possible solutions.
Could you tell me where to check for errors please?

**Comment by atm1504** [comment]
@arundhati24 @batbrain7 please merge it.

**Pull request merged**